targeted/search cvedetails using target server header.js

```diff
@@ -0,0 +1,11 @@
+// Captures Server header from the application response and searches cvedetails.com for known target server vulnerabilities.
+
+
+function invokeWith(msg) {
+
+		headers = msg.getResponseHeader().getHeader("Server")
```

Comment:
Consider renaming the variable to ```header```.
It should be checked that the header is defined. If response does not have the ```Server``` header it will be searched for "null".